src/IconToggle/index.js

```diff
@@ -1,89 +1,120 @@
 import React, { useState, useEffect } from 'react'
```

Comment:
```suggestion
import React from 'react'
```
Those are no longer being used, correct?

Reply:
correct